source/octf/fs/IFileSystemViewer.h

```diff
@@ -3,13 +3,36 @@
  * SPDX-License-Identifier: BSD-3-Clause-Clear
  */
 
-#ifndef SOURCE_OCTF_ANALYTICS_STATISTICS_IFILESYSTEMVIEWER_H
-#define SOURCE_OCTF_ANALYTICS_STATISTICS_IFILESYSTEMVIEWER_H
+#ifndef SOURCE_OCTF_FS_IFILESYSTEMVIEWER_H
+#define SOURCE_OCTF_FS_IFILESYSTEMVIEWER_H
 
 #include <string>
+#include <octf/proto/parsedTrace.pb.h>
 
 namespace octf {
 
+struct FileNodeId {
```

Comment:
I think we should keep only FileId structure. And it should be an input to the FS viewer.